### PR TITLE
DNS and crypto rule tweaks

### LINKED ISSUE #683: resolve DNS namespace
current namespace: host-interaction/network/dns/resolve

propose to move this under communication

### LINKED ISSUE #709: mine cryptocurrency
- string: "Steam" and potentially others appear to be too broad

### PR DESCRIPTION
**Pull request by mr-tz**
closes #683
closes #709

At least two minor updates this week 😄

### CODE REVIEW

File: nursery/reference-cryptocurrency-strings.yml [hunk 1]
```diff
@@ -20,7 +20,6 @@ rule:
       - string: "xmr."
       - string: "monero."
       - string: "Bitcoin"
-      - string: "Bitcoin"
```
Comment (mr-tz) [inline nursery/reference-cryptocurrency-strings.yml:23]: duplicate

File: nursery/reference-cryptocurrency-strings.yml [hunk 2]
```diff
@@ -44,7 +43,3 @@ rule:
       - string: "Monero"
       - string: "Graft"
       - string: "Zcash"
-      - string: "Ya.money"
-      - string: "Ya.disc"
-      - string: "Steam"
-      - string: "vk.cc"
```
Comment (mr-tz) [inline nursery/reference-cryptocurrency-strings.yml:50]: don't appear to be related to cryptocurrency and I couldn't identify where they came from originally

**Review by williballenthin — APPROVED** [review]
